mandatory/1-fix-functions.js

```diff
@@ -10,20 +10,19 @@
 
 */
 
-function mood() {
-  let isHappy = true;
-
-  if (isHappy) {
+function mood(isHappy) {
```

Comment:
Good job on this. It was a mistake in the homework, but it's great you fixed it yourself 👍 🌟